exporting/graphite/graphite.c

```diff
@@ -51,6 +51,60 @@ int init_graphite_instance(struct instance *instance)
     return 0;
 }
 
+/**
+ * Copy a label value and substitute underscores in place of charachters which can't be used in Graphite output
+ *
+ * @param dst a destination string.
+ * @param src a source string.
+ * @param len the maximum number of characters copied.
+ */
+
+void sanitize_graphite_label_value(char *dst, char *src, size_t len)
+{
+    while (*src != '\0' && len) {
+        if (isspace(*src) || *src == ';' || *src == '~')
+            *dst++ = '_';
+        else
+            *dst++ = *src;
+        src++;
```

Comment:
Do not we accept UTF-8 values for graphite?

Reply:
Yes, we accept. We don't touch any characters except spaces, `;`, and `~`.

Reply:
When we merge the PR that you already reviwed, I think you do not need to go every address, instead, you need to use the function that we already have inside Netdata to measure the size of the character to advance.

Reply:
It was discussed and a decision was made to keep checks for every character because otherwise there is a risk to access memory outside allocated boundaries.